docs/feature_flagging.md

```diff
@@ -0,0 +1,29 @@
+### Feature Flagging
+
+#### Step 1
+
+Create you feature toggle at https://unleash.artsy.net and enable it in the desired environment(s).
+
+#### Step 2
+
+To use feature flagging, simply import the useFeatureFlagging method and pass in the name of the feature toggle you created in step 1. This will enabled you to conditionally render your new feature.
```

Comment:
```suggestion
To use feature flagging, simply import the useFeatureFlag method and pass in the name of the feature toggle you created in step 1. This will enable you to conditionally render your new feature.
```